inc/configfield.class.php

```diff
@@ -104,6 +104,21 @@ static function install(Migration $migration) {
             $input["index"]                                = 0;
             $field->add($input);
          }
+
+         // Init date_last_generated
+         if (countElementsInTable($type::getTable())) {
+            $max = $DB->request([
+               'SELECT' => ['MAX' => 'date_creation as date'],
+               'FROM' => $type::getTable()
+            ])->current()['date'];
+
+            $cfield = new self();
+            $cfield->getFromDBByCrit(['itemtype' => $type]);
+            $cfield->update([
+               'id' => $cfield->getID(),
+               'date_last_generated' => $max
+            ]);
+         }
```

Comment:
To prevent computation of this value for all supported itemtypes on each plugin update:

```suggestion
         // Init date_last_generated
         $cfield = new self();
         if (
             $cfield->getFromDBByCrit(['itemtype' => $type])
             && $cfield->fields['date_last_generated'] === null
             && countElementsInTable($type::getTable())
         ) {
            $max = $DB->request([
               'SELECT' => ['MAX' => 'date_creation as date'],
               'FROM' => $type::getTable()
            ])->current()['date'];

            $cfield->update([
               'id' => $cfield->getID(),
               'date_last_generated' => $max
            ]);
         }
```